pkg/features/kube_features.go

```diff
@@ -1153,7 +1154,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	LoadBalancerIPMode: {Default: false, PreRelease: featuregate.Alpha},
 
-	ImageMaximumGCAge: {Default: false, PreRelease: featuregate.Alpha},
+	ImageMaximumGCAge: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
move this line so it is in alphabetical order in the list of features, that helps minimize conflicts